plugin.py

```diff
@@ -74,8 +74,6 @@ def install_or_update(cls) -> None:
                 for root, dirs, files in os.walk(os.path.join(tmp, "extension", "server", "bin", binplatform)):
                     for file in files:
                         os.chmod(os.path.join(root, file), 0o744)
-                # Make sure package storage path exists for new users
-                os.makedirs(cls.basedir(), exist_ok=True)
```

Comment:
Awesome! However, that line was added because if the user doesn't yet have a $CACHE/Package Storage directory, then installation will fail as well. I think the correct thing to do here is to change `cls.basedir()` into `cls.storage_path()`.